core/vm/contracts.go

```diff
@@ -23,6 +23,7 @@ import (
 	"fmt"
 	"math/big"
 
+	"github.com/cloudflare/circl/sign/bls"
```

Comment:
This is not available for me. Do they implement actual BLS verifcation on top of precompiles? Is this documented somewhere?

Reply:
The precompile is implemented by us. They simply implement BLS signatures.

Reply:
take a look here: https://pkg.go.dev/github.com/cloudflare/circl/sign/bls